### PR TITLE
CCD Apps Not Working Develop-to-Develop on Preview

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-8995

### Change description ###

Rename suppression file and audit only prod dependencies

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -48,8 +48,6 @@ withPipeline("angular", product, component) {
     enableAksStagingDeployment()
     disableLegacyDeployment()
 
-    before('securitychecks') {sh 'yarn test:audit'}
-
     after('checkout') {
```
Comment (MSancaktutar) [inline Jenkinsfile_CNP:51]: this shouldn't be removed before pipeline's own audit goes to prevention mode

File: ccd-yarn-audit-known-issues [hunk 2]
```diff
@@ -0,0 +1,3 @@
+https://npmjs.com/advisories/1179
+https://npmjs.com/advisories/1500
+https://npmjs.com/advisories/1544
```
Comment (danlysiak) [inline ccd-yarn-audit-known-issues:3]: I think we should fix these if possible. The first two I'm pretty sure have been fixed in the Node services; the third might be fixable in a similar manner.

File: yarn-audit-known-issues [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
...:false,"bundled":false},"advisory":{"findings":[{"version":"3.9.1","paths":["@hmcts/ccd-case-ui-toolkit>@angular/cli>pacote>npm-registry-fetch"]}],"id":1544,"created":"2020-07-07T19:05:44.803Z","updated":"2020-07-07T19:05:50.413Z","deleted":null,"title":"Sensitive Data Exposure","found_by":{"link":"","name":"Unknown","email":""},"reported_by":{"link":"","name":"Unknown","email":""},"module_name":"npm-registry-fetch","cves":[],"vulnerable_versions":"<4.0.5 || >=5.0.0 <8.1.1","patched_versions":">=4.0.5 <5.0.0 || >=8.1.1","overview":"Affected versions of npm-registry-fetch are vulnerable to an information exposure vulnerability through log files.   The package supports URLs like <protocol>://[<user>[:<password>]@]<hostname>[:<port>][:][/]<path>. The password value is not redacted and is printed to stdout and also to any generated log files.","recommendation":"Upgrade to version 4.0.5, 8.1.1 or later.","references":"- [GitHub Advisory](https://github.com/npm/npm-registry-fetch/security/advisories/GHSA-jmqm-f2gx-4fjv)","access":"public","severity":"low","cwe":"CWE-532","metadata":{"module_type":"","exploitability":3,"affected_components":""},"url":"https://npmjs.com/advisories/1544"}}}
```
Comment (danlysiak) [inline yarn-audit-known-issues:5]: Should be able to delete this file after fixing the issues.

File: ccd-yarn-audit-known-issues (hunk 2 above)
Comment (sumansreenivasan) [reply ccd-yarn-audit-known-issues]: Have fixed the vulnerabilities

File: yarn-audit-known-issues (hunk 3 above)
Comment (sumansreenivasan) [reply yarn-audit-known-issues]: Have fixed the vulnerabilities

**Pull request merged**